tests/components/upnp/conftest.py

```diff
@@ -210,6 +215,32 @@ async def mock_async_create_upnp_device(
         yield mock_async_create_upnp_device, mock_igd_device
 
 
+@pytest.fixture
+def mock_mac_address_from_host():
+    """Get mac address."""
+    with patch(
+        "homeassistant.components.upnp.config_flow.async_get_mac_address_from_host",
```

Comment:
We should patch the library instead of patching Home Assistant

untested, but I *think* `homeassistant.components.upnp.device.get_mac_address` should work here